### DISCUSSION TITLE
DNS Error

### DISCUSSION THREAD

**Discussion by Sherop777** [iotaledger/firefly#8017] [Help & Support]
Hello!
I haven't gotten into my Firefly in years. My version is 1.3.3.

I try to send my funds but I get a DNS error

"ClientError: `error sending request for url (https://chrysalis-nodes.iota.org/api/v1/info): error trying to connect: dns error: Host desconocido. (os error 11001)`"

Do you know of any DNS to add manually?

I don't want to update because I forgot my seed. I only remember the pin number and password. Additionally I have the file "firefly-backup-2022-03-08T18-02-37.stronghold"

Thanks!

**Comment by begonaalvarezd** [comment]
Yo ushould update to the latest version, the network was upgraded in October 2023 so that version you are running is not compatible anymore with the network, the latest version as of today is V2.0.6. You can download the latest version here https://firefly.iota.org/

**Reply by begonaalvarezd** [reply]
If you dont remember your seed, before upgrading just make a stronghold backup that you can use to restore at ny time or you can use that one you are mentioning you already have. if you dont have your 24 words I suggest you create a new profile, safely save your new 24 words and move your funds there, you shouldnt rely on stronghold only

**Reply by begonaalvarezd** [reply]
Lastly, when you upgrade, you might see less balance than you had before, just run the balance finder until you see all your funds, you need to let the new firefly find your addresses. You can search by going to settings > advanced > balance finder
